tests/python/test_errors.py

```diff
@@ -321,8 +366,43 @@ def test_bad_type_run_script(use_cluster, context):
     with pytest.raises(TypeError):
         c.run_script(key, fn_name, inputs, 42)
 
+@pytest.mark.skipif(
+    not test_gpu,
+    reason="SMARTREDIS_TEST_DEVICE does not specify 'gpu'"
+)
```

Comment:
Since all executions in this test case trigger type errors, we don't have to skip this one on a non-GPU system